bindings/packager/nodejs/src-ts/plugins/electron/index.ts

```diff
@@ -0,0 +1,211 @@
+import type { Config, Resource } from "../../config";
+import type { PackageJson } from "..";
+import fs from "fs-extra";
+import path from "path";
+import os from "os";
+import { download as downloadElectron } from "@electron/get";
+import extractZip from "extract-zip";
+import { Pruner, isModule, normalizePath } from "./prune";
+
+export default async function run(
+  appPath: string,
+  packageJson: PackageJson
+): Promise<Partial<Config> | null> {
+  let electronPath;
+  try {
+    electronPath = require.resolve("electron", {
+      paths: [appPath],
+    });
+  } catch (e) {
+    return null;
+  }
+
+  const userConfig = packageJson.packager || {};
+
+  const electronPackageJson = JSON.parse(
+    (
+      await fs.readFile(
+        path.resolve(path.dirname(electronPath), "package.json")
+      )
+    ).toString()
+  );
+
+  const zipPath = await downloadElectron(electronPackageJson.version);
```

Comment:
can't we reuse the one in `node_modules`?

Reply:
nope, this downloads the prebuilt electron zip, not the NPM dependency

Reply:
doesn't the NPM dependency also come with a prebuilt binary? anyways I guess it is fine to download but shouldn't we at least cache it?

Reply:
the @electron/get package caches it by default

Reply:
ah nice